Kuroba/app/src/main/java/com/github/adamantcheese/chan/core/site/common/taimaba/TaimabaActions.java

```diff
@@ -0,0 +1,146 @@
+/*
+ * Kuroba - *chan browser https://github.com/Adamantcheese/Kuroba/
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.github.adamantcheese.chan.core.site.common.taimaba;
+
+import com.github.adamantcheese.chan.core.site.SiteAuthentication;
+import com.github.adamantcheese.chan.core.site.common.CommonSite;
+import com.github.adamantcheese.chan.core.site.common.MultipartHttpCall;
+import com.github.adamantcheese.chan.core.site.http.DeleteRequest;
+import com.github.adamantcheese.chan.core.site.http.DeleteResponse;
+import com.github.adamantcheese.chan.core.site.http.Reply;
+import com.github.adamantcheese.chan.core.site.http.ReplyResponse;
+import org.jsoup.Jsoup;
+
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import okhttp3.HttpUrl;
+import okhttp3.Response;
+
+import static android.text.TextUtils.isEmpty;
+
+public class TaimabaActions extends CommonSite.CommonActions {
+    public TaimabaActions(CommonSite commonSite) {
+        super(commonSite);
+    }
+
+    @Override
+    public void setupPost(Reply reply, MultipartHttpCall call) {
```

Comment:
The reason why you can't post is probably because a bunch of these methods are constructing the wrong sort of request. You should use a browser to see what network traffic happens during a request so you can perform the right things in these methods.